pwiz_tools/Shared/Common/SystemUtil/DiffAttribute.cs

```diff
@@ -0,0 +1,66 @@
+﻿/*
+ * Original author: Tobias Rohde <tobiasr .at. uw.edu>,
+ *                  MacCoss Lab, Department of Genome Sciences, UW
+ *
+ * Copyright 2018 University of Washington - Seattle, WA
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+
+namespace pwiz.Common.SystemUtil
+{
+    public interface IAuditLogObject
```

Comment:
Can we come up with a name for this interface which is more descriptive of what it does, rather than where it's being used?
Yes, the audit log needs to know how to describe things, but do we want to call the interface "IAuditLogObject", or should it be maybe "IDescribable".